### DISCUSSION TITLE
How to use (non programmer)

### DISCUSSION THREAD

**Discussion by madbrickykd9uqe** [google/filament#6613] [Q&A]
Hey folks, I'm a ham radio tech trying to get a 3d model of antennas and their rf near field radiation rendered from sketch, photos, or output from my antenna design program (EZNEC screenshot)
I'm not a programmer or even slightly skilled in running these types of applications directly in code.
Can you direct me on how to learn to do what you do or if it's possible to simply install and run the program.
Like I said I'm an old electronics nerd and my skills are in a different realm but I'm always trying to learn new stuff.
I have an HP G3 Studio workstation with Windows 11 and this Samsung s20fe for the application.
Thank you kindly, Craig in Chicago KD9UQE

**Comment by romainguy** [comment]
Hi Craig,

Unfortunately Filament requires programming skills to be used. It's not a standalone application.
